rslearn/dataset/window.py

```diff
@@ -13,6 +13,47 @@
 
 logger = get_logger(__name__)
 
+LAYERS_DIRECTORY_NAME = "layers"
```

Comment:
I would recommend adding some tests for these directory structure functions

Reply:
added a few tests. some of them will just throw error if we change the directory structure, while the one for mark_layer_completed/is_layer_completed will actually test logic.